lib/delayed/backend/iron_mq_backend.rb

```diff
@@ -1,18 +1,18 @@
 require_relative 'iron_mq_config'
 
 module Delayed
-  class Worker
+  class IronMqBackend
     class << self
       attr_accessor :config, :ironmq, :queue_name, :delay, :timeout, :expires_in,
```

Comment:
Before it was only one default queue for all tasks, now `task.perform_later(queue: 'test_queue')` will run task in `test_queue` and user should specify this queue in the queue list `Delayed::IronMqBackend.queues = ['test_queue', ...]`